src/redux/books/books.js

```diff
@@ -1,55 +1,45 @@
 import { v4 as uuidv4 } from 'uuid';
-import defaultBooks from './default-books';
+
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/V825UBB3aPe1HQTJEANF/books/';
 
 // Actions
 const LOAD = 'bookstore/books/LOAD';
-const CREATE = 'bookstore/books/CREATE';
-const UPDATE = 'bookstore/books/UPDATE';
-const REMOVE = 'bookstore/books/REMOVE';
```

Comment:
- [ ] [OPTIONAL] I don't think this was a good idea to delete all the action types(CREATE, REMOVE) and only remain with `LOAD` because remember action types actually describe what we need to do, but for now if we need to create a new book we dispatch the action which is called `LOAD` or when we need to delete the a book we dispatch the same action `LOAD` which is not really a good practice, it could have been better to keep the action types you had before since they all describe actions that are being performed 😉